src/gwt/src/org/rstudio/studio/client/events/ReticulateEvent.java

```diff
@@ -70,8 +76,9 @@ protected void dispatch(Handler handler)
    public static final Type<Handler> TYPE = new Type<>();
 
    // synchronize with SessionReticulate.R
-   public static final String TYPE_PYTHON_INITIALIZED = "python_initialized";
+   public static final String TYPE_PYTHON_INITIALIZED = "pythAon_initialized";
```

Comment:
Is this change intended?

Reply:
It definitely is not -- thanks for catching that.